README.md

```diff
@@ -7,15 +7,27 @@ TermIt docker serves to spin off a TermIt deployment, consisting of :
 
 ## Prerequisities
 First, You should have Docker & Docker Compose installed (and accessible under current user).
-Second, You need a valid GraphDB Free installation package (or a valid SE or EE license).
 Third, You need to have docker images of [termit](https://github.com/kbss-cvut/termit-ui) and [termit-server](https://github.com/kbss-cvut/termit) built.
 
-Edit `docker-compose.yml`:
-- download GraphDB Free ZIP and place it into the `graphdb` folder.
-- set `services.termit-db-server.build.args.GRAPHDB_ZIP` to the respective GraphDB distribution file, e.g. `graphdb-free-9.6.0-dist.zip`
-- set `services.termit.image` to the respective docker image of [TermIt frontend](https://github.com/kbss-cvut/termit-ui)
-- set `services.termit-server.image` to the respective docker image of [TermIt backend](https://github.com/kbss-cvut/termit)
-- adjust `ROOT` in .env to reflect the local context prefix the app will be running on.
-- adjust `URL` in .env to reflect the server the app will be running on.
-
-To run TermIt, execute `docker-compose up` 
+## Running TermIt
+1. Download [GraphDB](https://www.ontotext.com) (Free, SE, EE) standalone server ZIP archive and place it into the 'db-server' folder. TermIt can be successfully run with GraphDB Free 9.10.1, but other versions should be usable as well.
+2. Set `GRAPHDB_FILE` variable in `.env to the name of the file You just downloaded.
+3. Clone [TermIt server](https://github.com/kbss-cvut/termit) as a new subfolder `termit-server`, i.e.
+   
+   `git clone https://github.com/kbss-cvut/termit termit-server`
+4. Clone [TermIt UI](https://github.com/kbss-cvut/termit-ui) as a new subfolder `termit`, i.e.
+   
+   `git clone https://github.com/kbss-cvut/termit-ui termit`
+5. Clone [Annotace](https://github.com/kbss-cvut/annotace) (`termit` branch) as a new subfolder `annotace-server`, i.e.
+   
+   `git clone https://github.com/kbss-cvut/annotace annotace-server --branch termit`
```

Comment:
Why is it now necessary to clone annotace, when previously we had a Spark-based image published?

Reply:
Or actually, why is it necessary to clone both TermIt and TermIt UI as well? I thought we had them published as Docker images as well.

Reply:
As I wrote in the title of https://github.com/kbss-cvut/termit-docker/issues/2, my goal was to have the docker orchestration self-contained. 

I extended the instructions how to use pre-built images if available. Also note that there are no pre-built TermIt docker images for this TermIt version.